maistra/ci/common.sh

```diff
@@ -48,3 +51,10 @@ function bazel_test() {
 
 # Fix path to the vendor deps
 sed -i "s|=/work/|=$(pwd)/|" maistra/bazelrc-vendor
+
+# Create symbolic links to existent executables in the builder
+mkdir -p $WORKDIR/maistra/local/bin
```

Comment:
OK

Reply:
@jwendell is there a good shell script linting tool that you can recommend? (I bet that will come in handy for me as well as I start out with 2.4). This may help catching things like these before review?

Reply:
Yes, we use *ShellCheck* in istio and operator. We would just need to setup this lint job for proxy and envoy. Setting it up comes with the cost of eventually fix any lint error in all .sh files, not only ours.

I'd recommend install it locally and run it against our maistra directory. I even have a plugin for vscode that does the lint in real time as I edit the file.